src/Scheduler/Cache.php

```diff
@@ -33,20 +35,22 @@ public function getMinutesNotInCache(): array
         return $this->_scheduleMinutesNotInCache;
     }
 
-    protected function _isMinuteScheduledInCache(\DateTime $referenceMinuteDateTime): bool
+    protected function _isMinuteScheduledInCache(\DateTime $referenceMinuteDateTime) : bool
     {
         $isMinuteScheduled = false;
         $referenceMinuteDateTimeString = $referenceMinuteDateTime->format(self::DATE_TIME_FORMAT_CACHE_MINUTE);
         $cacheItemPool = $this->_getCacheItemPoolRepository()->getById(self::CACHE_ITEM_POOL_ID);
-        $hasItem = $cacheItemPool->hasItem(self::CACHE_SCHEDULED_AHEAD_KEY_PREFIX . $referenceMinuteDateTimeString);
+        $hasItem = $cacheItemPool->hasItem(
+            $this->getLockPrefix() . self::CACHE_SCHEDULED_AHEAD_KEY_PREFIX . $referenceMinuteDateTimeString
```

Comment:
@alexberryman - what is the method signature of `hasItem`? I'm curious if we can name a getter for `$this->getLockPrefix() . self::CACHE_SCHEDULED_AHEAD_KEY_PREFIX . $referenceMinuteDateTimeString`